exec/memory/base/src/main/java/io/netty/buffer/DrillBuf.java

```diff
@@ -882,4 +882,71 @@ public void print(StringBuilder sb, int indent, Verbosity verbosity) {
     }
   }
 
+  // The "unsafe" methods are for use ONLY by code that does its own
+  // bounds checking. They are called "unsafe" for a reason: they will crash
+  // the JVM if values are addressed out of bounds.
+
+  /**
+   * Write an integer to the buffer at the given byte index, without
+   * bounds checks.
+   *
+   * @param index byte (not int) index of the location to write
+   * @param value the value to write
+   */
+
+  public void unsafePutInt(int index, int value) {
```

Comment:
The first argument in these unsafePutXXX methods is an offset right? Should the 'index' be changed to an 'offset' ?

Reply:
Fixed.